root/types.js

```diff
@@ -43,30 +76,67 @@ declare type EntityT = {
 declare type InstrumentT =
   & EntityT
   & {
-    +description: string,
```

Comment:
The `+` there marked the property as covariant, which basically makes it read-only (you can't do `instrument.description = blah;`): https://flow.org/blog/2016/10/04/Property-Variance/

I think it makes sense to keep, and it should be added to other properties that should be read-only (if not all, nearly all).